pkg/controller/aws/s3/s3bucket.go

```diff
@@ -195,7 +197,6 @@ func (r *Reconciler) _create(bucket *bucketv1alpha1.S3Bucket, client s3.Service)
 
 	// Set access keys into a secret for local access creds to s3 bucket
 	secret := connectionSecret(bucket, accessKeys)
-	secret.OwnerReferences = append(secret.OwnerReferences, bucket.OwnerReference())
```

Comment:
Note this is the line that caused https://github.com/crossplaneio/crossplane/issues/360. `connectionSecret()` also adds an owner reference, so we end up with two.